_docs/policies/000-submitting-policy.md

```diff
@@ -0,0 +1,36 @@
+# Purpose
+
+Policy Zero establishes the foundational guidelines for creating, reviewing, and maintaining policies in the `status-go` GitHub repository. This policy aims to create a collaborative, inclusive, and transparent process for defining repository policies, specifically regarding how developers engage with and contribute to the repository.
+
+# Submitting a Policy Proposal
```

Comment:
I believe it would be helpful to include the justification for the given policy as well.

- A policy MUST include a brief justification, addressing the question: "Why has this policy been introduced?"

Reply:
I actually added this to my latest pending commit.